tasks.py

```diff
@@ -592,6 +592,14 @@ def pylint(context, target=None, recursive=False):
         run_command(context, command)
 
 
+@task
+def docs_local(context):
+    """Runs local instance of mkdocs serve (ctrl-c to stop)."""
+    command = "mkdocs serve"
+    context.nautobot.local = True
+    run_command(context, command)
```

Comment:
Perhaps something like:

```suggestion
    if is_truthy(context.nautobot.local):
        run_command(context, "mkdocs serve")
    else:
        start(context, service="mkdocs")
```